src/content/admin/dashboard.mdx

```diff
@@ -205,3 +202,9 @@ From this page you can follow a simple step-by-step guide to get the major compo
 If you choose to pull images from a private registry in addition to [Okteto's Container Registry](core/container-registry.mdx), you can configure credentials for that service here.
 
 [Review this page for a full description of how to configure and start using your own Registry Credentials →](admin/registry-credentials/index.mdx)
+
+## Cloud Credentials
```

Comment:
Should we also add a "Registry Credentials" section in this doc?

Reply:
Yes, I'll add in a separate PR.